web/src/stream_popover.js

```diff
@@ -212,9 +213,21 @@ function build_stream_popover(opts) {
                 $(e.currentTarget).hide();
                 e.stopPropagation();
             });
-
-            new ClipboardJS($popper.find(".copy_stream_link")[0]).on("success", () => {
-                popover_menus.hide_current_popover_if_visible(instance);
+            // copy the link to the stream
+            $popper.on("click", ".copy_stream_link", () => {
+                const $streamLinkElement = $(".copy_stream_link ");
+                const formatedLinkText = `
+                <a href=${$streamLinkElement.data("clipboard-text")}>${try_stream_topic_syntax_text($streamLinkElement.data("clipboard-text")).replaceAll(/\**/g, "\u200B")}</a>
+               `;
+                const clipboardItem = new ClipboardItem({
+                    "text/plain": new Blob([$streamLinkElement.data("clipboard-text")], {
```

Comment:
Rather than reading `$streamLinkElement.data("clipboard-text")` from the DOM multiple times, just store it in a variable and use it in both places.